src/ResourceManagement/HDInsight/HDInsight.Tests/Helpers/GetClusterSpecHelpers.cs

```diff
@@ -22,6 +22,8 @@ namespace HDInsight.Tests.Helpers
 {
     public static class GetClusterSpecHelpers
     {
+        private const string ADLClusterRootPath = "/Clusters/SDK";
```

Comment:
Waiting for confirmation from @shefaliv, so that I can include the changes (if any) in the upcoming PR.